src/sql/workbench/contrib/tasks/browser/media/tasksPanel.css

```diff
@@ -7,22 +7,27 @@
 	padding: 10px 22px 0 22px;
 }
 
+.task-history .monaco-tree > .monaco-scrollable-element > .monaco-tree-wrapper {
+	overflow-x: auto !important;
```

Comment:
do you really need the important modifier? are you trying to overwrite the overflow-x set directly on the element?

Reply:
The important is needed because overflow is set to hidden on the element.